code/modules/client/preferences_savefile.dm

```diff
@@ -654,12 +654,14 @@
 	if(!S)
 		return FALSE
 	S.cd = "/loadouts"
-	var/json_loadout_manager = ""
-	READ_FILE(S["loadouts_manager"], json_loadout_manager)
-	if(!json_loadout_manager)
-		return FALSE
-	loadout_manager = jatum_deserialize(json_loadout_manager)
-	return !isnull(loadout_manager)
+	var/loadout_version = 0
+	READ_FILE(S["loadout_version"], loadout_version)
+	if(loadout_version != CURRENT_LOADOUT_VERSION)
+		return list()
```

Comment:
Why dont you do a conversion from v5 to v6 here

Reply:
yeah i'm just lazy, i'll do that